auth-web/src/util/constants.ts

```diff
@@ -411,22 +411,13 @@ export enum Permission {
 
 export enum LDFlags {
     AffiliationInvitationRequestAccess = 'enable-affiliation-invitation-request-access',
-    AuthLearnMore = 'auth-options-learn-more',
     BannerText = 'banner-text',
     BusSearchLink = 'bus-search-staff-link',
-    DisableGovNAccountCreation='disable-govn-account',
     EnableBcCccUlc = 'enable-bc-ccc-ulc',
     EnableBusinessNrSearch = 'enable-business-nr-search',
     EnableDetailsFilter = 'enable-transactions-detail-filter',
     EnableEFTPaymentMethod = 'enable-eft-payment-method',
-    EnableFasDashboard = 'enable-fas-dashboard',
```

Comment:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/861c2c2b-79c0-40e4-a2fa-425348426ae2)

Reply:
All  your LD screenshots are for Prod, right?

Reply:
Yes